Modules/Devices/src/Devices.Domain/DomainErrors.cs

```diff
@@ -51,6 +51,11 @@ public static DomainError DeletionProcessMustBeInStatus(DeletionProcessStatus de
         return new DomainError($"error.platform.validation.device.deletionProcessMustBeInStatus{deletionProcessStatus}", $"The deletion process must be in status '{deletionProcessStatus}'.");
     }
 
+    public static DomainError IdentityCannotBeDeleted()
+    {
+        return new DomainError($"error.platform.validation.device.identityCannotBeDeleted", $"The identity cannot be deleted because it has not been approved or is not past due approval.");
```

Comment:
> it has not been approved or is not past due approval

"it" gramatically refers to the identity, which doesn't make sense, because an identity can neither be approved, not can it be past due approval